src/libmeasurement_kit/ooni/web_connectivity.cpp

```diff
@@ -139,9 +139,14 @@ static void compare_http_requests(SharedPtr<nlohmann::json> entry,
     }
 }
 
-static void compare_dns_queries(SharedPtr<nlohmann::json> entry,
+static void compare_dns_queries(mk::http::Url url, SharedPtr<nlohmann::json> entry,
                                 std::vector<std::string> experiment_addresses,
                                 nlohmann::json control, Settings options) {
+    // Short circuit for the case where we're using an IP address.
+    if (mk::net::is_ip_addr(url.address)) {
+        (*entry)["dns_consistency"] = "consistent";
```

Comment:
I guess this is a reasonable value, however we should mention it in the spec. I added a note to: https://github.com/ooni/spec/issues/148#issuecomment-520373921

Reply:
Thanks!